service/service.go

```diff
@@ -159,13 +60,13 @@ func (s *Service) getDependency(dependencyKey string) (*Dependency, error) {
 	return nil, fmt.Errorf("dependency %s do not exist", dependencyKey)
 }
 
-// validateConfigurationEnv checks presence of env variables in mesg.yml under env section.
-func (s *Service) validateConfigurationEnv(env map[string]string) error {
+// ValidateConfigurationEnv checks presence of env variables in mesg.yml under env section.
+func (s *Service) ValidateConfigurationEnv(env map[string]string) error {
```

Comment:
Same comment, this one could be quite ok because it doesn't have any impact on the data but could be moved to the deploy api